### ISSUE TITLE
release-please-action not tagging releases after `manifest-pr` is merged

### ISSUE THREAD

**Issue by whizzzkid** [google-github-actions/release-please-action#719]
### TL;DR

Here is an implementation of creating a release PR: https://github.com/ipfs/ipfs-companion/blob/main/.github/workflows/ci.yml#L96-L104

This creates the [release PR](https://github.com/ipfs/ipfs-companion/pull/1157) as expected. On merging this PR, this [action runs again](https://github.com/ipfs/ipfs-companion/actions/runs/4187553788/jobs/7257678686), but does nothing. The expectation is not to create a release, because the need is to add more assets to the created release. The [release step](https://github.com/ipfs/ipfs-companion/blob/main/.github/workflows/ci.yml#L106-L151) is separate and is executed when a new tag gets pushed. However release-please-action does not push the tag hence this step never gets executed.

### Expected behavior

Since release-please-action is only being used to create a collector PR and not the release itself, the expectation as per [the documentation](https://github.com/google-github-actions/release-please-action#whats-a-release-pr) was:

> When you're ready to tag a release, simply merge the release PR.

This does not happen if the release step is not happening. 

### Observed behavior

The release PR merges without creating a release tag, breaking the workflow.

### Action YAML

```yaml
release-pr:
    runs-on: ubuntu-latest
    needs: [test]
    if: github.ref == 'refs/heads/main' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch')
    steps:
      - uses: google-github-actions/release-please-action@v3.7.3
        with:
          command: manifest-pr
          changelog-notes-type: github
```

**Comment by chingor13** [comment]
If you want it to tag, use the `manifest` command. The `manifest-pr` command only creates the PR (`manifest` does PR creation and tagging)
